woocommerce/woocommerce-ios
# Unit tests fail for non-US regions

Fixes #1550 

## Steps to reproduce
0. Check out the `develop` branch.
1. Set your simulator Settings app to use Chinese (Traditional) and set the region to Hong Kong.
2. Run the unit tests locally on Xcode.

## Testing
Do Not rely on CI for this. Tests pass in CI because the Xcode region (probably) defaults to the US.

0. Check out this branch.
1. Set your simulator Settings app to use Chinese (Traditional) and set the region to Hong Kong.
2. Run the unit tests locally on Xcode.

Expectation: 8 unit tests in `DateWooTests` fail. These remaining 8 failing unit tests are expected to pass once the `NSLocalizedString`s have been translated.

Update release notes:
